build/bom/pom.xml

```diff
@@ -378,21 +378,6 @@
                 <version>${version.infinispan}</version>
             </dependency>
 
-            <dependency>
-                <groupId>org.infinispan</groupId>
-                <artifactId>infinispan-spring5-common</artifactId>
-                <version>${version.infinispan}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.infinispan</groupId>
-                <artifactId>infinispan-spring5-embedded</artifactId>
-                <version>${version.infinispan}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.infinispan</groupId>
-                <artifactId>infinispan-spring5-remote</artifactId>
-                <version>${version.infinispan}</version>
-            </dependency>
             <dependency>
```

Comment:
I think `infinispan-spring-boot-starter-embedded` and `infinispan-spring-boot-starter-remote` can be removed